workers/unity/Packages/com.improbable.gdk.core/Commands/WorldCommands.cs

```diff
@@ -98,13 +118,40 @@ public static Request CreateRequest(Entity entity, EntityId? entityId = null, ui
                 };
             }
 
+            /// <summary>
+            ///     An object that represents the response of a CreateEntity command from the SpatialOS runtime.
+            /// </summary>
             public struct ReceivedResponse
             {
+                /// <summary>
+                ///     The status code of the command response. Can be used to check whether the command succeeded and
```

Comment:
Rather than "can be used..." I think this should be explicit. Along the lines "If equal to StatusCode.Success then the command succeeded."
Strictly it can only be used to check for success if it succeeded. Otherwise the check might be wrong (can succeed and get a failure status).